scout/server/blueprints/variants/controllers.py

```diff
@@ -636,10 +636,13 @@ def observations(store, loqusdb, case_obj, variant_obj):
     obs_data['cases'] = []
     institute_id = variant_obj['institute']
     for case_id in obs_data.get('families', []):
-        if case_id != variant_obj['case_id'] and case_id.startswith(institute_id):
-            other_variant = store.variant(variant_obj['variant_id'], case_id=case_id)
+        if case_id != variant_obj['case_id']:
+            # other case might belong to same institute, collaborators or other institutes
             other_case = store.case(case_id)
-            obs_data['cases'].append(dict(case=other_case, variant=other_variant))
+            # display only variants which belong to the same institute or collaborators
+            if other_case and (other_case.get('owner') == institute_id or institute_id in other_case.get('collaborators', [])):
```

Comment:
Yep, that looks like the old functionality. Now, it would be really nice to also pay attention to what institutes the current user has access to. It is not unreasonable that you can use also "your own" patient data to determine if the variant is of interest. The actual user institute should be available from store.user() something, shouldn't it?

Reply:
Good point, and of course we should include the option in this PR. I don't want to touch it again later!